src/test/scala/uk/gov/hmrc/perftests/example/UknwAuthCheckerApiRequests.scala

```diff
@@ -20,27 +20,25 @@ import io.gatling.core.Predef._
 import io.gatling.http.Predef._
 import io.gatling.http.request.builder.HttpRequestBuilder
 import io.netty.handler.codec.http.HttpResponseStatus
+import org.scalacheck.Gen
 import uk.gov.hmrc.performance.conf.ServicesConfiguration
+import wolfendale.scalacheck.regexp.RegexpGen
 
 object UknwAuthCheckerApiRequests extends ServicesConfiguration {
 
   val baseUrl: String             = baseUrlFor("uknw-auth-checker-api")
   val route: String               = "/authorisations"
   private val bearerToken: String = if (runLocal) s"$${accessToken}" else s"Bearer $${accessToken}"
-
-  val navigateToHomePage: HttpRequestBuilder =
-    http("Navigate to Home Page")
-      .get(s"$baseUrl$route/vat-return-period")
-      .check(status.is(200))
-      .check(css("input[name=csrfToken]", "value").saveAs("csrfToken"))
+  val eoriGen: Gen[String]        = RegexpGen.from("^(GB|XI)[0-9]{12}|(GB|XI)[0-9]{15}$")
```

Comment:
This won't work. We have dedicated stub records we allow in our service. Coordinate with Alex to get one that can work with the happy path.

Reply:
Refactored the code to have payload as static data.